src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -139,7 +139,13 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
             marginVertical: 12
           }}
         />
-        <Text style={styles.bioText}>{addDotAndRemoveNewline(bio || '')}</Text>
+        {!bio ? (
+          <Text style={styles.bioTextNull}>
+            What should others message you about? Add your bio and conversation starters here.
+          </Text>
+        ) : (
+          <Text style={styles.bioText}>{addDotAndRemoveNewline(bio || '')}</Text>
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering for the bio text is a good addition. However, it's important to ensure that `addDotAndRemoveNewline` function handles null or undefined values correctly in case `bio` is not provided.
```diff
- <Text style={styles.bioText}>{addDotAndRemoveNewline(bio || '')}</Text>
+ <Text style={styles.bioText}>{bio ? addDotAndRemoveNewline(bio) : ''}</Text>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->